solutions/swb-reference/integration-tests/support/utils/utilities.ts

```diff
@@ -7,6 +7,14 @@ import { v4 as uuidv4 } from 'uuid';
 import { DEFAULT_POLLING_INTERVAL_SECONDS, DEFAULT_POLLING_MAX_WAITING_SECONDS } from './constants';
 import HttpError from './HttpError';
 
+/**
+ * Marks all properties (including nested) as Partial.
+ * https://stackoverflow.com/questions/47914536/use-partial-in-nested-property-with-typescript
+ */
+type RecursivePartial<T> = {
```

Comment:
Could you be more descriptive in tsdocs as to why this is needed/preferred as compared to previous types?

Reply:
That's being removed